jsonnet/telemeter/rules.libsonnet

```
@@ -204,6 +204,22 @@
                 max by (_id, managed_cluster_id) (acm_managed_cluster_worker_cores:max)
               |||,
             },
+            {
+              // Hypershift Cluster vCPU-hours for the last hour.
+              // The divisor of scalar(count_over_time(vector(1)[1h:5m])) allows us to get an effective average value having "absent data" treated as 0.
+              // sum(...) by (_id) is used ensure a single datapoint per cluster ID.
+              record: 'hostedcluster:hypershift_cluster_vcpus:vcpu_hours',
```

(suggestion) "...vcpus:vcpu..." feels redundant. Also https://prometheus.io/docs/practices/rules/#naming
```suggestion
              record: 'hostedcluster:hypershift_cluster_vcpus:avg1h',
```

---

Naming `hostedcluster:hypershift_cluster_vcpus:vcpu_hours` recording rule that way has been requested in [OSD-25491](https://issues.redhat.com/browse/OSD-25491). I agree that the name is not ideal (repeating the `vcpu` part...); that said it is consistent with the other recording rules in the file "averaging" CPU consumption during the last hour.
Also, naming this recording rule that way is quite useful to make sure that `rosa:cluster:vcpu_hours` recording rule, based on it, is not comparing apples and oranges.
I am not against changing this suffix, but if we do it or start to do it, lets make sure it is consistently changed everywhere.

---

Yeah, I have started a thread with the author of the JIA ticket.
We should eventually consider refactoring those rules from time to time and better name the rules ans the metrics.